libdevcore/RangeMask.h

```diff
@@ -71,6 +71,16 @@ class RangeMask
 		return ret;
 	}
 
+	/// @returns a range containing at most _items smallest elements.
```

Comment:
`@returns a range that starts with the first element and ends @a _items later or at the first excluded element`?